### DISCUSSION TITLE
No mail delivery after install

### DISCUSSION THREAD

**Discussion by KrystianPlichta** [modoboa/modoboa#2426] [Installer]
Hi, i used installer for my instalation modoboa, and i can send mails, but when i try to reply. Modoboa tells me that user dont exist.

for the purpose of pasting the log I changed the domain

> git clone https://github.com/modoboa/modoboa-installer
> cd modoboa-installer
> sudo ./run.py test.domain.pl

LOG : 
https://pastebin.com/6tvTXjVQ

**Comment by rdk187** [comment]
Hi all,
could anybody shed some light on that case?
I have exactly the same problem, my modoboa can send but not receive emails. I can see in the logs that the user does not exist although he exists.

Thank you!

**Comment by KrystianPlichta** [comment]
the problem has been solved, the problem was that the hostname of the virtual machine was the same as the name of the domain I wanted to add. In the official postfix documentation it says that the hostname must be different from the domain name added in postfix
